spec/models/solidus_content/entry_spec.rb

```diff
@@ -13,10 +13,28 @@
     let(:post) { create(:entry_type, name: :post, provider_name: :raw) }
     let!(:post_one) { create(:entry, slug: 'one', options: {title: "first post"}, entry_type: post)}
 
-    it "returns data for type and slug" do
-      expect(
-        SolidusContent::Entry.data_for(:post, 'one')
-      ).to eq({title: "first post"})
+    context "with using correct type and slug" do
```

Comment:
👍 can you squash [this commit](https://github.com/nebulab/solidus_content/pull/29/commits/094cd02989ca3ac18d0b59e1f360c0dc10badea9) to the previous one? This way we'll have the change along with its specs together, and also fix the comments that slipped into the commit message 😜